website/content/en/docs/installation/cap-operator-manager/kyma-cluster.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Kyma Cluster"
+linkTitle: "Kyma Cluster"
+weight: 10
+type: "docs"
+tags: ["setup"]
+description: >
```

Comment:
Would be great if instead of this page content the top level link "Kyma Cluster" could directly redirect to the link below.
Right now users need two additional clicks if they start from the top.

Reply:
Added direct link to the page